java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -451,6 +451,10 @@ class ReferenceModeStore private constructor(
   private fun newBackingInstance(): CrdtModel<CrdtData, CrdtOperationAtTime, Referencable> =
     crdtType.createCrdtModel()
 
+  /** Gets data from the Backing Store with the corresponding [referenceId] */
+  suspend fun getLocalData(referenceId: String) =
```

Comment:
Can this be private?

Reply:
Tests use it so I've left it as public, but I'll add a comment.